packages/next-auth/tests/jwt.test.ts

```diff
@@ -0,0 +1,23 @@
+import { SessionStore } from "../src/core/lib/cookie"
+
+it("should result in the correct cookie order", async () => {
+  const cookieName = "__Secure-next-auth.session-token"
+  const logger = console
+  const sessionStore = new SessionStore(
```

Comment:
This should be expanded to have 11 chunks to ensure the string re-ordering works as expected.